### ISSUE TITLE
Add role=alert on error message

### ISSUE THREAD

**Issue by pascalbleuse-tech** [Orange-OpenSource/Orange-Boosted-Bootstrap#3688]
### Prerequisites

- [x] I have [searched](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed feature requests
- [x] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/.github/CONTRIBUTING.md)

### Proposal

Hello,

Currently, on OUDS, error handling in forms is designed to work on input, and messages are triggered correctly during vocalization.

However, if we want validation on field blur, the messages are no longer vocalized because the role="alert" attribute is missing from the error element.

For a11y compatibility, it would be good to add this tag in the OUDS web version and in the documentation.

Thank you.

### Motivation and context

Increase a11y compatibility

**Comment by MaxLardenois** [comment]
Thank you for the issue, we will look into it with our a11y experts
